pkgs/development/ocaml-modules/atd/default.nix

```
@@ -1,14 +1,13 @@
-{ lib, atdgen-codec-runtime, menhir, easy-format, buildDunePackage, which, re, nixosTests }:
+{ lib, atdgen-codec-runtime, menhir, easy-format, buildDunePackage, which, re, yojson, nixosTests }:
```

```suggestion
{ lib, atdgen-codec-runtime, menhir, easy-format, buildDunePackage, re, yojson, nixosTests }:
```

`which` was removed.

---

Fixed. Thanks.